type_data/tables/lm_trap.yaml

```diff
@@ -2,5 +2,5 @@ name:
   - (?:green-tinted|thick glass|clear glass) vial
   - (?:light violet|emerald|reddish-brown|wavering|icy blue|stone grey|amethyst|dark red) sphere
   - dark crystal
-  - steel needle
-  - steel jaws
\ No newline at end of file
+  - slender steel needle
+  - pair of small steel jaws
```

Comment:
@ondreian Should this be a migration instead?

Reply:
I think not, since this is in the core (previously working) set of definitions?

I want to use migrations for anything we are layering on top that currently does not exist in GameObj (like SoS, Red Forest)

Reply:
:+1: makes sense to me